readme/configuration-management.md

```diff
@@ -70,14 +70,39 @@ We need to find a better way of preventing this than manually monitoring module
 
 ## Configuration Split workflow
 
-BLT recommends using the Config Split module to manage configuration on most projects. For a description of how to use Config Split, see this excellent [blog post by Jeff Geerling](https://www.jeffgeerling.com/blog/2017/adding-configuration-split-drupal-site-using-blt-and-acquia-cloud).
+### Overview
+
+BLT recommends using the Config Split module to manage configuration on most projects. For an overview of how to use Config Split, see this excellent [blog post by Jeff Geerling](https://www.jeffgeerling.com/blog/2017/adding-configuration-split-drupal-site-using-blt-and-acquia-cloud).
```

Comment:
For backwards compatibility, BLT currently defaults to using features. We could instead default to config split and introduce an update hook to set the cm.strategy to "features" for projects created before 8.7.*. Thoughts?